src/gui/guimain.cpp

```diff
@@ -125,23 +125,17 @@ extern "C" void gui_run(void) {
     //Screens::Init(ScreenFactory::Screen<screen_splash_data_t>);
     Screens::Init(screen_initializer, screen_initializer + (sizeof(screen_initializer) / sizeof(screen_initializer[0])));
 
+    //TIMEOUT variable getting value from EEPROM when EEPROM interface is inicialized
+    GuiDefaults::menu_timeout_enabled = (bool)variant_get_ui8(eeprom_get_var(EEVAR_MENU_TIMEOUT));
```

Comment:
it is better to write `variant_get_ui8(eeprom_get_var(EEVAR_MENU_TIMEOUT)) != 0;` instead of an explicit C-style cast `(bool)`